tools/packaging/kata-deploy/examples/ngnix-deployment-dragonball.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment-dragonball
```

Comment:
The filename is wrong, you are using `ngnix` instead of `nginx`

Reply:
fixed